static/client/components/Navigation/NavigationElement/NavigationElement.tsx

```diff
@@ -56,6 +58,45 @@ const NavigationElement = ({ activePageName, page, project, onSelect }: INavigat
     }
   }, [activePageName, page.name]);
 
+  const statusIcon = useCallback(() => {
```

Comment:
Can you make it a separate component instead? I think logically this fits more into a component, than to a callback function